js/clickup-add-issues/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "clickup-add-issues",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "prepack": "PACKAGES_DIR=js node ../../scripts/publish.js",
+    "build:refapps": "tsc -p tsconfig.json",
+    "postbuild:refapps": "yarn prepack && yarn packseq",
+    "packseq": "PACKAGES_DIR=js node ../../scripts/packsequence.js",
+    "clean": "rm -rf ./dist .bic_cache"
+  },
+  "keywords": [],
+  "author": "Scramjet <open-source@scramjet.org>",
+  "license": "ISC",
+  "dependencies": {
+    "@scramjet/types": "^0.22.0",
```

Comment:
bit old, 0.33.4

Reply:
i was unable to update to a newer version, getting following error
![image](https://user-images.githubusercontent.com/53794300/233071520-18b22b93-8409-46fe-95bc-bf7f7c5ec674.png)

Reply:
0.33.5 has been released today. You have a typo in screenshot above. 0.34.4 instead of suggested 0.33.4

Reply:
Thanks for pointing out, resolved now :D